src/components/controls/controls.js

```diff
@@ -0,0 +1,218 @@
+var registerControls = require('../../core/controls').registerControls;
+var THREE = require('../../lib/three');
+
+var MAX_DELTA = 0.2; // ms
+var PI_2 = Math.PI / 2;
+
+/**
+ * Controls component.
+ *
+ * Receives input events from device-specific control components, and applies movement and rotation
+ * to the element accordingly.
+ */
+module.exports.Component = registerControls('controls', {
+  dependencies: ['velocity'],
+
+  schema: {
+    enabled: { default: true },
+
+    flyingEnabled: { default: false },
+
+    position: { default: ['gamepad-controls', 'keyboard-controls', 'touch-controls'] },
```

Comment:
I just have a couple cardboards here. I'm theoretically in the March 28th shipping group, but got delayed until April 12th-ish. Thanks, might be helpful if one of you wants to take a look at attaching `hmd-controls::getVelocity()`, otherwise I hopefully don't have too much longer to wait.